core/lib/types/src/snapshots.rs

```diff
@@ -193,6 +193,25 @@ pub struct SnapshotRecoveryStatus {
     pub l1_batch_root_hash: H256,
     pub miniblock_number: MiniblockNumber,
     pub miniblock_root_hash: H256,
-    pub last_finished_chunk_id: Option<u64>,
-    pub total_chunk_count: u64,
+    pub storage_logs_chunks_processed: Vec<bool>,
+}
+
+// Used only in tests
+#[derive(Debug, PartialEq)]
+pub struct InitialWriteDbRow {
```

Comment:
I've seen those methods, but I think that using them would introduce too much logic in the tests and make the tests less readable enough to justify introducing dedicated methods

Reply:
IMO, testing that "real" DAL methods work when invoked with snapshot-related inputs is better than creating dedicated methods for testing purposes, but that's a personal opinion. Curious to see what's @RomanBrodetski's opinion on the matter. If we go with the current approach, please place `*Row` structs in the DAL crate; e.g., you can replace `DBStorageLog` because it doesn't seem to be used anywhere.